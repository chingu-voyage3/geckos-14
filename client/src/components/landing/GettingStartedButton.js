import React from 'react';
import { Link } from 'react-router-dom';

const GettingStartedButton = () => {
  return (
    <div className="welcome-text-button">
      <Link to="/getstarted"><button type="button" className="btn btn-outline-secondary btn-lg">Get Started</button></Link>
    </div>
  );
};

export default GettingStartedButton;
