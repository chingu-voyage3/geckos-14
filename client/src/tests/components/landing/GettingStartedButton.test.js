import React from 'react';
import GettingStartedButton from '../../../components/landing/GettingStartedButton';

import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<GettingStartedButton />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
