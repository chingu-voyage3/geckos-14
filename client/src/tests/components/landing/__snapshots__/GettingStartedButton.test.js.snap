// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="welcome-text-button"
>
  <Link
    replace={false}
    to="/getstarted"
  >
    <button
      className="getstarted-button"
      type="button"
    >
      Get Started
    </button>
  </Link>
</div>
`;
